<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainwindow</class>
 <widget class="QDialog" name="Mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
	Copyright 2013 Emanuel Claesson

	Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
	you may not use this file except in compliance with the License.
	You may obtain a copy of the License at

		http://www.apache.org/licenses/LICENSE-2.0

	Unless required by applicable law or agreed to in writing, software
	distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
	See the License for the specific language governing permissions and
	limitations under the License.
*/

/*
	COLOR_DARK     = #191919
	COLOR_MEDIUM   = #353535
	COLOR_MEDLIGHT = #5A5A5A
	COLOR_LIGHT    = #DDDDDD
	COLOR_ACCENT   = #3D7848
*/

* {
	background: #191919;
	color: #DDDDDD;
	
}

QPushButton {
color: Black;
background-color: #c4c4c4;
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 15px;
padding-left: 5px;
padding-right: 5px;

}
QPushButton:hover {
	background: white;
}

QTableView
{
color: Black;
background-color: white;
border-width: 1px;
/*border-color: #339;*/
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 15px;
font-color:black;
}</string>
  </property>
  <widget class="QPushButton" name="pb_retour">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ajouter">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_modifier">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_supprimer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>530</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mailing</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabplace">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>621</width>
     <height>401</height>
    </rect>
   </property>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>281</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>15</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>7</number>
    </property>
    <property name="bottomMargin">
     <number>7</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_id"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_cin"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_nom"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_prenom"/>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_email"/>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Nombre de bus</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_nbp"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Nombre de reservation</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="lineEdit_nbr"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QDateTimeEdit" name="edit_td"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>621</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_show">
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Client fid√©le</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
